credits: update links to the full list of code and docs contributors (#6111)
pingcap/docs: credits.md

# Credits

> Contributors play an important role in the robust development of TiDB. We sincerely appreciate all contributors who submit code, write and translate documents for TiDB.

Reviewer: ```suggestion
Each contributor has played an important role in promoting the robust development of TiDB. We sincerely appreciate all contributors who have submitted code, written and translated documents for TiDB.
```

## TiDB developers

> TiDB developers contribute to new feature development, performance optimization, and bugfixes. The following is the list of contributors to TiDB related repo:

Reviewer: ```suggestion
TiDB developers contribute to new feature development, performance improvement, and stability The following is the list of contributors in TiDB related repos:
```

Reviewer 2: @TomShawn Did you remove the `.` on purpose?

Shouldn't that be `stability improvements` instead of `stability`? What about other bugfixes that are not related to stability?

Reviewer: Sorry, that was not on purpose. This is a literal translation from the Chinese version: https://github.com/pingcap/docs-cn/pull/5901/files#diff-a3cfe79e912417e309dc5273eb5b3ef6b8c123f8f597286e1ea9ce12f7afbc6eR13. I have added bug fixes here. Later we will synchronize it back.

> TiDB developers contribute to new feature development, performance optimization, and stability guarantee. The following is the list of contributors to TiDB related repo:

Reviewer 2: ```suggestion
TiDB developers contribute to new feature development, performance optimization, and bugfixes. The following is the list of contributors to TiDB related repo:
```

> TiDB developers contribute to new feature development, performance improvement, stability guarantee, and bug fixes. The following is the list of contributors in TiDB related repos:

Reviewer: @en-jin19 Please help sync it back to the Chinese version, according to @dveeden's comment. Thanks!

- [pingcap/tiup](https://github.com/pingcap/tiup/graphs/contributors)
- [pingcap/br](https://github.com/pingcap/br/graphs/contributors)
- [pingcap/dm](https://github.com/pingcap/dm/graphs/contributors)
- [pingcap/tidb-binlog](https://github.com/pingcap/tidb-binlog/graphs/contributors)
- [pingcap/tidb-dashboard](https://github.com/pingcap/tidb-dashboard/graphs/contributors)
- [pingcap/ticdc](https://github.com/pingcap/ticdc/graphs/contributors)
- [pingcap/tidb-tools](https://github.com/pingcap/tidb-tools/graphs/contributors)
- [pingcap/tidb-lightning](https://github.com/pingcap/tidb-lightning/graphs/contributors)
- [pingcap/tispark](https://github.com/pingcap/tispark/graphs/contributors)
- [pingcap/dumpling](https://github.com/pingcap/dumpling/graphs/contributors)
- [tikv/client-java](https://github.com/tikv/client-java/graphs/contributors)
- [tidb-incubator/TiBigData](https://github.com/tidb-incubator/TiBigData/graphs/contributors)
- [ti-community-infra](https://github.com/orgs/ti-community-infra/people)

For the full list of contributors, see [SIG | TiDB DevGroup](https://contributor.tidb.io/sig).

> ## Writers and translators of TiDB documentation

Reviewer: ```suggestion
## Writers and translators for TiDB documentation
```

> Writers and translators contribute to writing and translating documents for TiDB related projects. The following is the list of contributors to TiDB documentation related repo:

Reviewer: ```suggestion
Writers and translators write and translate documents for TiDB and the related projects. The following is the list of contributors in TiDB documentation related repos:
```
